doc/factory/custom_vars.html

```diff
@@ -1701,6 +1701,32 @@ <h3><a name="frontend_vars"></a>Frontend Client Variables</h3>
               </p>
             </td>
           </tr>
+          <tr>
+            <td><b>GLIDEIN_BLACKHOLE_NUMJOBS</b></td>
+            <td>Int</td>
+            <td>
+              <p>
+                This attribute represents number of jobs completed too quickly
+                before raising the blackhole detection flag. More details in
+                blackhole detection section, on the bottom of who webpage
+                (Lifetime of a Glidein).
+              </p>
+            </td>
+          </tr>
+          <tr>
+            <td><b>GLIDEIN_BLACKHOLE_RATE</b></td>
+            <td>Int</td>
+            <td>
+              <p>
+                The variable indicates if the glidein have been detected as a
+                blackhole based on the stats obtained from the STARTD. The stats
+                are always the same for all slots as it’s a global statistic
+                about the whole startd. By default is False. More details in
+                blackhole detection section, on the bottom of who webpage
+                (Lifetime of a Glidein).
```

Comment:
Rate at which a Glidein is flagged as a black hole and retired. If a Glidein consumes more than GLIDEIN_BLACKHOLE_RATE jobs per second, then it will become a black hole, and stop accepting new jobs. 
When undefined or set to 0, the black hole mechanism is disabled.
See the <a href="#blackhole">blackhole detection section</a> below for more details.